base/applications/mscutils/servman/lang/fr-FR.rc

```diff
@@ -157,6 +157,7 @@ BEGIN
             TVS_LINESATROOT | TVS_DISABLEDRAGDROP, 8, 151, 236, 68
     LTEXT "Certains services dépendent d'autres services, pilotes système et groupes d'ordre de chargement. Si un composant système est arrêté ou ne fonctionne pas correctement, les services dépendants peuvent être affectés.", IDC_STATIC, 8, 7, 238, 36
     LTEXT "Ce service dépend des composants système suivants", IDC_STATIC, 8, 57, 236, 9
+    LTEXT "The following components depend on this service", IDC_STATIC, 8, 140, 236, 9
```

Comment:
```suggestion
    LTEXT "Les c&omposants système suivants dépendent de ce service", IDC_STATIC, 8, 140, 236, 9
```

Reply:
Added including the ending colon. Thanks.